mindmeld/auto_annotator.py

```diff
@@ -764,33 +825,68 @@ def _resolve_person(self, entity):
         Returns:
             entity (dict): A resolved entity dict.
         """
-        entity["dim"] = self.ANNOTATOR_TO_DUCKLING_ENTITY_MAPPINGS[entity["dim"]]
+        entity["dim"] = ANNOTATOR_TO_SYS_ENTITY_MAPPINGS[entity["dim"]]
 
-        if len(entity["body"]) >= 2 and entity["body"][-2:] == "'s":
-            entity["value"] = {"value": entity["body"][:-2]}
-            entity["body"] = entity["body"][:-2]
-            entity["end"] -= 2
+        if self.language == ENGLISH_LANGUAGE_CODE:
+            if len(entity["body"]) >= 2 and entity["body"][-2:] == "'s":
```

Comment:
These two conditionals can be combined into one.

Reply:
Done